app/views/govuk_component/related_items.raw.html.erb

```diff
@@ -0,0 +1,34 @@
+<%
+  sections ||= []
+%>
+<aside class="govuk-related-items" role="complementary">
+  <% sections.each do |section| %>
+    <% section_title = section[:title] || "Elsewhere on GOV.UK" %>
```

Comment:
Is that a [regression vs the old behaviour](https://github.com/alphagov/static/blob/master/app/views/root/related.raw.html.erb#L7-L34)? If not, then I think it's not a blocker, unless fixing it required a significant API change - like passing "Elsewhere" (internal, and [external](https://github.com/alphagov/static/blob/master/app/views/root/related.raw.html.erb#L37-L51) links as a separate arg?